### ISSUE TITLE
conflict with AE2?

### ISSUE THREAD

**Issue by AmonRay** [cleannrooster/Spellblade-Next#9]
https://gist.github.com/AmonRay/cd13ae3bed9e9c4014bc1149c331dd8b

everything works on spellbladenext-fabric-1.0.23.2.jar, issue comes up with 1.0.23.3

**Comment by cleannrooster** [comment]
That's strange, because the mixin indicated by the crash log hasn't been changed in many versions. Did you update a mod called Gigeresque at the same time? It seems that that mod has something to do with it as well.

**Comment by AmonRay** [comment]
yep, both mods has been updated. But as I mentioned: everything works with spellbladenext-fabric-1.0.23.2.jar :( Should I duplicate issue on Gigeresque github?
